Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -63,12 +67,39 @@ public MixedRealityCameraProfile CameraProfile
                 return cameraProfile;
             }
         }
+        
+        /// <inheritdoc/>
+        public Camera Main
+        {
+            get
+            {
+                if (main == null || !main.isActiveAndEnabled)
+                {
+                    FindOrCreateMainCamera();
+                }
+                return main;
+            }
+        }
 
         private DisplayType currentDisplayType;
         private bool cameraOpaqueLastFrame = false;
+        private static Camera main;
 
         /// <inheritdoc />
         public override void Initialize()
+        {
+            FindOrCreateMainCamera();
+
+            // Ensure that the camera is parented under the mixed reality playspace
+            main.transform.SetParent(MixedRealityPlayspace.Transform);
+
```

Comment:
In some scenarios, such as spectator view, cameras may already have parent transforms influencing their positions. If a MixedRealityCameraSystem is dynamically added/initialized, I think this will break those experiences.

It seems like we actually need to set the mixed reality playspace transform as the parent of any main camera parent transforms. This should probably be a search up the parenting hierarchy compared to a replacement.

Reply:
Let's start an issue where we track these experience requirements and how our current setup breaks them. We can use it to discuss whether custom camera system implementations can accommodate them / whether changes need to be made to the camera cache / playspace.